pkg/cmd/server/origin/handlers.go

```diff
@@ -21,27 +21,35 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// RootPaths is the list of root paths we serve under "/". We expose it here so that we can
+// access it from integration tests
+var RootPaths = unversioned.RootPaths{}
```

Comment:
We'd need to create a field and accessor method on the internal `MasterConfig` - is this OK?